lib/cucumber/blendle/steps/resource_steps.rb

```diff
@@ -59,6 +59,20 @@
   step %(the client does a #{method} request to "#{url}" with the following content:), content
 end
 
+# * And the client does a POST request to the "user_item_preference" resource with user_id "bartsimpson" and item_id "bnl-telegraaf-20130212-1" and the following content:
+#   """json
+#   {
+#     "prefer": "less"
+#   }
+#   """
+#
+When(/^the client does a (POST|PUT) request to the "([^"]*)" resource with (\S+) "([^"]*)" and (\S+) "([^"]*)" and the following content:$/) do |method, resource, key, value, key2, value2, content|
+  api  = HyperResource.from_body(api_body)
+  url  = api.send(resource, key => value, key2 => value2).href
```

Comment:
That results in something different (is equivalent to `{ :key => value, :key2 => value2 }`, which is not what we want here)